src/components/TinderStyleVoting/index.tsx

```diff
@@ -32,13 +37,13 @@ const VotingCards = () => {
 	const currentIndexRef = useRef(currentIndex);
 	const [tempActiveProposals, setTempActiveProposals] = useState([]);
 	const [skippedProposals, setSkippedProposals] = useState<number[]>([]);
+	const [swipeDirection, setSwipeDirection] = useState<string | null>(null);
 
 	const childRefs: any = useMemo(
 		() =>
```

Comment:
what's that?

Reply:
there is a overlay card, which had green, red and yellow color as per card swipe. so for that this state is being used